plugins/importexport/native/filter/NativeXmlIssueFilter.inc.php

```diff
@@ -74,8 +73,21 @@ function handleElement($node) {
 		$issue->setCurrent($node->getAttribute('current'));
 		$issue->setAccessStatus($node->getAttribute('access_status'));
 
-		$issueDao->insertObject($issue);
-		$deployment->setIssue($issue);
+		$foundIssue = null;
+		foreach ($node->getElementsByTagName('issue_identification') as $n) {
+			$searchIssue = $issueDao->newDataObject();
+			$this->parseIssueIdentification($n, $searchIssue, false);
```

Comment:
+1 to this.  I'm thinking too hierarchically, and too oriented to backwards compatibility.  This would be a step in the right direction.